### PR TITLE
buwugs

### PR DESCRIPTION
**Pull request by aymendirar**
add space, close format paren

### CODE REVIEW

File: src/commands/uwu-ify.ts [hunk 1]
```diff
@@ -34,8 +34,9 @@ const uwuify: Command = {
 				binding_info.str +
 				`\n` +
 				`  `.repeat(binding_info.line_number) +
-				`(format "${"~a ".repeat(binding_info.line_number).trim()}"` +
+				`(format "${"~a ".repeat(binding_info.line_number).trim()}" ` +
```
Comment (braxtonhall) [inline src/commands/uwu-ify.ts:37]: Do you need the space if you remove the trim?

Comment (aymendirar) [reply src/commands/uwu-ify.ts:37]: i want the trim before the `"` but the space after it so that it looks like `(format "~a ~a" uwu1 uwu2)`

Comment (aymendirar) [reply src/commands/uwu-ify.ts:37]: otherwise it would look like `(format "~a ~a "uwu1 uwu2)`?

Comment (braxtonhall) [reply src/commands/uwu-ify.ts:37]: trU

Comment (braxtonhall) [reply src/commands/uwu-ify.ts:37]: where are the unit test!

**Review by braxtonhall — APPROVED** [review]

**Pull request merged**